src/components/parallax-scroll.component.js

```diff
@@ -30,34 +24,37 @@ type State = {
   parallaxHeaderHeight: number,
 };
 
-const styles = StyleSheet.create({
-  background: {
-    position: 'absolute',
-    top: 0,
-    backgroundColor: colors.primaryDark,
-  },
-  stickySection: {
-    height: STICKY_HEADER_HEIGHT,
-    backgroundColor: colors.primaryDark,
-    alignItems: 'center',
-    justifyContent: 'flex-end',
-  },
-  stickySectionText: {
-    color: colors.white,
-    ...fonts.fontPrimaryBold,
-    fontSize: normalize(16),
-    margin: 10,
-  },
-  fixedSectionLeft: {
-    position: 'absolute',
-    bottom: 0,
-  },
-  fixedSectionRight: {
-    position: 'absolute',
-    bottom: 10,
-    right: 10,
-  },
-});
+const Background = styled.View`
+  position: absolute;
+  top: 0;
+  background-color: ${colors.primaryDark};
+  ${props => props.style};
```

Comment:
@shmesa22 after a second read, as `style` now only contain the height, could you rename the prop to `height`, and use it in this style? (`height: ${props => props.height}`)

I think it would be more clear